bob/src/link_manager.rs

```diff
@@ -21,28 +24,63 @@ impl LinkManager {
     }
 
     async fn checker_task(factory: Factory, nodes: Arc<[Node]>, period: Duration) {
+        let now = coarsetime::Clock::now_since_epoch().as_secs();
```

Comment:
There is no need to use `coarsetime`, because measurement will be called once in 100ms